packages/types/organizations.ts

```diff
@@ -1,23 +1,26 @@
 import { z } from "zod";
 
-export const ZSubscriptionStatus = z.enum(["active", "cancelled", "inactive"]).default("inactive");
-
-export type TSubscriptionStatus = z.infer<typeof ZSubscriptionStatus>;
-
-export const ZSubscription = z.object({
-  status: ZSubscriptionStatus,
-  unlimited: z.boolean().default(false),
+export const ZOrganizationBillingPlan = z.enum(["free", "startup", "scale", "enterprise"]);
```

Comment:
can we / should be use our constants here instead of `free`, `scale`,...

Reply:
I dont think we can use it here, our constants are in the `packages/lib` package and the types packages does not have access to it.